apps/ecs/behaviours/check-validation.js

```diff
@@ -37,6 +37,14 @@ module.exports = superclass => class extends superclass {
       }
     }
 
+    if(key === 'worker-zipcode') {
+      const zipCode = req.form.values[key];
+      const zipCodeRegex = '^[a-z0-9][a-z0-9\\- ]{0,10}[a-z0-9]$';
+      if(zipCode && !validators.url(zipCode) && !validators.regex(zipCode, zipCodeRegex)) {
```

Comment:
seems like `!validators.url(zipCode)` validation is redundant and will be covered by zipCodeRegex check

Reply:
I would suggest bringing in the `maxlength` validation rule for `worker-zipcode`, which is defined in `fields/index.js`
because currently the `zipCodeRegex` check happens before the `maxlength` check.

Reply:
Done with this change